tests/time_tests/test_runner/.automation/tgl_test_config.yml

```diff
@@ -0,0 +1,48 @@
+- device:
+    name: CPU
+  model:
+    path: ${KMB_MODELS_PKG}/resnet-50-pytorch/caffe2/FP16/resnet-50-pytorch.xml
```

Comment:
please name it VPUX_MODELS_PKG

Reply:
Also for the other commit, the device is -d VPUX

Reply:
Thanks, fixed